go/pkg/apis/recognizer/language_recognizer.go

```diff
@@ -106,12 +108,24 @@ func extractExtensions(paths []string) map[string]int {
 	return extensions
 }
 
-func getFilePaths(root string) ([]string, error) {
+func GetFilePaths(root string) ([]string, error) {
 	var files []string
-	err := filepath.Walk(root,
+	ignoreFile, errorIgnoreFile := getIgnoreFile(root)
```

Comment:
Seems the Go convention is to start with an uppercase letter so should'nt it be the case here ?